src/Uploadcare/Widget.php

```diff
@@ -86,6 +105,25 @@ public function getIntegrationData()
         return $integrationData;
     }
 
+    /**
+     * Enable signed uploads.
+     *
+     * @param array $attributes Widget attributes.
+     * @return array Array with attributes.
+     */
+    private function withSignatureAttributes($attributes)
```

Comment:
why `with`?

Reply:
Because it sounds good : ) `$this->withSignatureAttributes`. Why not?

Reply:
`addSignagureAttributes` makes more sense here

Reply:
^